gui/erroritem.h

```diff
@@ -76,7 +76,7 @@ class ErrorItem {
     QString message;
 };
 
-Q_DECLARE_METATYPE(ErrorItem);
+Q_DECLARE_METATYPE(ErrorItem)
```

Comment:
does this cause any warning/error message? For tools (Cppcheck) the extra semicolon can make the code clearer - it can be seen that this has nothing to do with the class declaration below..